packages/docs/docs/docs/upgrading.md

```diff
@@ -160,6 +160,24 @@ truffle(local)> myContract.x().then(x => x.toString())
 43
 ```
 
+Now, just as a mental exercise, let's imagine that instead of just adding a new 
+function to the contract (a change to functionality), we wanted to add a new 
+variable `t` (a change to storage layout). How do we set the initial value of `t`?
+The variables `x` and `s` were initialized with the `initialize` function,
+which was called when the proxy was created via the `zos create MyContract --init initialize --args ...` 
+command. The `update` command also accepts `--init` and `--args` parameters, so we can use a function
+to initialize the new variable once again. We cannot to use the same `initialize` function again, because
+the `Initializable` modifier guards it against being called more than once. We need a new function. A good name for the 
+new function could be something like `initializeVersion2` or `migrateToV2`. This function would simply
+set the initial value of `t` and should be called with `zos update MyContract --init migrateToV2 --args 99`.
+
+```
+function migrateToV2(uint256 _t) public {
+  require(_t == 0);
```

Comment:
Mention that this trick only works if t cannot be zero in this contract, otherwise it could be reinitialized